app.py

```diff
@@ -12,13 +11,16 @@ def index():
 
 @app.route('/video')
 def video():
+    print "Video request"
+
     FNULL = open(os.devnull, 'w')
 
-    hri_cmd = ['python', 'heart_rate_deamon.py', video_file, opencv_path]
-    opencv_ps = subprocess.Popen(hri_cmd, stdout=subprocess.PIPE, stderr=FNULL)
+    client_proc_cmd = 'python heart_rate_client.py ' + video_file  
```

Comment:
Thought you would ask. Basically, we need to write the output of the message queue to the ffmpeg process. I tried doing this with a generator but it expects a pipe (file) so I couldn't. 

There were 2 ways I thought of doing this: 

1) have this in a function that is run in another thread and is constantly writing to the ffmpeg process or 

2) have it in another process where I can just hook up the processes output into ffmpeg input. 

I didn't try 1 but I suspected there would be issues with the global interpreter lock that Python has (which means any I/O doesn't benefit from mulithreading I believe). I was thinking I would try this approach once I got streaming from YouTube to work and see if I sped up the output streaming.